svthalia/concrexit
# Move Event.slide to Slide.event

Closes #2759.



### Summary
This moves a ForeignKey to a OneToOne in the opposite direction.
After this, events only depends on announcements by including `SlideSerializer` in the event serializers.

### How to test
Steps to test the changes you made:
1. Check that the event screen still shows the right slide header if that was set before migrating.
2. Check that the events API still works.
